### ISSUE TITLE
 Port number ended with 'e'

### ISSUE THREAD

**Issue by lifeblood** [laravel/nova-issues#3330]
composer update 

```
[Composer\Downloader\TransportException]                                                                                                                          
  curl error 3 while downloading https://xxxxxx@gmail.com:93e5scOSDxa9bA1kaNUyBWMBVHbjU0Fsb8hZ0EBF@nova.laravel.com/packages.json: Port number ended with 'e'
```

why?

**Comment by crynobone** [comment]
Can you share the return output of `composer diagnose`?

**Comment by crynobone** [comment]
Seem to be fixed #3337
